### DISCUSSION TITLE
Class using github: all accounts blocked. IP blacklisted?

### DISCUSSION THREAD

**Discussion by Gunstick** [community/community#41613] [GitHub Education]
Hi

we stated a new coding class at the school.
It involves creating github accounts.

So there were 44 accounts created.

And today they are all blocked.
Creating a new account via mobile and then using it with the school IP gets it blocked as well.

I hope mine will now not be blocked too!

How do I contact github about this issue?

**Comment by radar-blip** [comment]
You can try one of the categories here: [Support Page](https://support.github.com/request)

Here is a better link actually: https://support.github.com/contact/education

**Comment by samus-aran** [comment]
Hi @Gunstick 👋 

We're happy you're here!

Support would be the best way to reach out to request re-instating those student accounts but I would also recommend that you also apply with GitHub as a new [GitHub Global Campus Partner School](https://education.github.com/schools) so that your institution can be labeled as a verified school.

**Reply by Gunstick** [reply]
Yes, will do the campus thing.
But meanwhile how to proceed with the re-instating. Can I do that in a big drop of all accounts emails or does every user need to do that on it's own (which is a hassle as they are just starting to learn programming. Many are computer illiterate)

**Reply by samus-aran** [reply]
Awesome to your school planning to sign up 🎉 

Regards to the students accounts GitHub support should be able to re-instate multiple accounts if found appropriate using information like email (academic) domain or IP and the date when the accounts were created etc.
